### ISSUE TITLE
The ability to favorite video players

### ISSUE THREAD

**Issue by epicEaston197** [Neos-Metaverse/NeosPublic#2535]
## Is your feature request related to a problem? Please describe.
[NA]

## Relevant issues
[NA.]

## Describe the solution you'd like
so like a keyboard or Avatar you would be able to favorite video players the same way 
when you favorite the video player that will become your default video player.

## Describe alternatives you've considered
[NA.]

## Additional context
when somebody hits ctrl-v to Paste
 a video instead of it being the default video player it will be the video player that they customized.

**Comment by ProbablePrime** [comment]
Thanks for your suggestion, This might be covered here: https://github.com/Neos-Metaverse/NeosPublic/projects/2#card-61036839

But it is unclear at the moment as this card doesn't currently include this UI type.

Thank you!

**Comment by Frooxius** [comment]
Yes this would be covered as well. The general goal is to make any common UI favoritable.

**Comment by ProbablePrime** [comment]
Thanks Froox, I'll update the card for you!
